addons/modules/stringtable.xml

```diff
@@ -1904,6 +1904,22 @@
             <Japanese>アクション文章</Japanese>
             <Korean>동작 텍스트</Korean>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleCreateIntel_ActionSound">
+            <English>Action Sound</English>
+            <German>Aktionsgeräusch</German>
+        </Key>
+        <Key ID="STR_ZEN_Modules_ModuleCreateIntel_LaptopKeyboard">
+            <English>Laptop Keyboard</English>
```

Comment:
Maybe for both of these keyboards - Keyboard (Laptop) / Keyboard (PC)?

This makes it quicker to see they are almost the same.

Reply:
Nah, I would leave it as it is.